### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by evoevodin**
Fixes integration tests.
Adds connection checks before running tests.

@skabashnyuk please review

### CODE REVIEW

File: wsmaster/codenvy-hosted-integration-tests/codenvy-integration-jpa-dao/src/test/java/WaitConnectionListener.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        // Tries to establish connection 10 times with 1 second delay
+        boolean isAvailable = false;
+        for (int i = 0; i < 10 && !isAvailable; i++) {
+            try (Connection connection = DriverManager.getConnection(dbUrl, dbUser, dbPassword)) {
+                isAvailable = true;
+            } catch (SQLException x) {
+                LOG.warn("An attempt to connect to the database failed with an error: {}", x.getLocalizedMessage());
+            }
+            try {
+                Thread.sleep(1000);
```
Comment (benoitf) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-jpa-dao/src/test/java/WaitConnectionListener.java]: TimeUnit.SECONDS.sleep(1); ?

File: wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/src/test/java/com/codenvy/api/tck/JpaIntegrationTckModule.java [hunk 2]
```diff
@@ -101,16 +106,39 @@
...
+        // Tries to establish connection 10 times with 1 second delay
+        boolean isAvailable = false;
+        for (int i = 0; i < 10 && !isAvailable; i++) {
+            try (Connection connection = DriverManager.getConnection(dbUrl, dbUser, dbPassword)) {
+                isAvailable = true;
+            } catch (SQLException x) {
+                LOG.warn("An attempt to connect to the database failed with an error: {}", x.getLocalizedMessage());
+            }
+            try {
+                Thread.sleep(1000);
```
Comment (benoitf) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/src/test/java/com/codenvy/api/tck/JpaIntegrationTckModule.java]: What about TimeUnit.SECONDS.sleep(1) ?

**Comment by benoitf** [comment]
funny to merge without replying to comments

**Comment by evoevodin** [comment]
i changed `Thread.sleep` -> `TimeUnit.SECONDS.sleep(1)`, did i have to say _Okay_ on all of them?

File: wsmaster/codenvy-hosted-integration-tests/codenvy-integration-jpa-dao/src/test/java/WaitConnectionListener.java (hunk 1 above)
Comment (evoevodin) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-jpa-dao/src/test/java/WaitConnectionListener.java]: Thank you, i will change it

File: wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/src/test/java/com/codenvy/api/tck/JpaIntegrationTckModule.java (hunk 2 above)
Comment (evoevodin) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/src/test/java/com/codenvy/api/tck/JpaIntegrationTckModule.java]: Thank you, i will changed it

**Comment by benoitf** [comment]
@evoevodin I would say at least one comment. not all of them

**Pull request merged**